ui/app/src/views/ViewProject.tsx

```diff
@@ -11,33 +11,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { useParams } from 'react-router-dom';
-import { Box, Container, Paper, Stack, Typography } from '@mui/material';
+import { useNavigate, useParams } from 'react-router-dom';
+import { Box, Button, Container, Paper, Stack, Typography } from '@mui/material';
 import { ErrorAlert, ErrorBoundary } from '@perses-dev/components';
 import FolderPound from 'mdi-material-ui/FolderPound';
 import ViewDashboard from 'mdi-material-ui/ViewDashboard';
+import { useState } from 'react';
 import { useDashboardList } from '../model/dashboard-client';
 import DashboardList from '../components/DashboardList';
+import CreateDashboardDialog from '../components/CreateDashboardDialog/CreateDashboardDialog';
+import { useSnackbar } from '../context/SnackbarProvider';
 
 interface RenderDashboardInProjectProperties {
   projectName: string;
 }
 
 function DashboardPageInProject(props: RenderDashboardInProjectProperties) {
+  const { infoSnackbar } = useSnackbar();
+  const navigate = useNavigate();
+
+  const [openCreateDashboardDialogState, setOpenCreateDashboardDialogState] = useState(false);
+
   const { data } = useDashboardList(props.projectName);
   if (data === undefined) {
     return null;
   }
+
+  const triggerDashboardCreation = function (name: string) {
+    navigate(`/projects/${props.projectName}/dashboards/${name}/create`);
+    infoSnackbar(`In order to create a new dashboard. You need to add at least one panel!`);
+  };
+
   return (
     <Paper>
       <Box p={1}>
-        <Stack direction="row" alignItems="center" gap={1} my={2}>
-          <ViewDashboard />
-          <Typography variant="h3">Dashboards</Typography>
+        <Stack direction="row" alignItems="center" justifyContent="space-between">
+          <Stack direction="row" alignItems="center" gap={1} my={2}>
+            <ViewDashboard />
+            <Typography variant="h3">Dashboards</Typography>
+          </Stack>
+          <Button variant="contained" size="small" onClick={() => setOpenCreateDashboardDialogState(true)}>
+            ADD DASHBOARD
```

Comment:
nit: better to use `textTransform: 'uppercase'` for accessibility

Reply:
```suggestion
            Add Dashboard
```

Reply:
For consistency with other parts of the app

Reply:
Accepting suggestions here can break the DCO when you go to merge, so might want to re-make the changes @cndonovan suggested as a normal signed commit locally btw

Reply:
Sure, I will do the change